packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -57,25 +57,28 @@ const CheckoutPage = props => {
         setPaymentInformationDone,
         resetReviewOrderButtonClicked,
         handleReviewOrder,
-        reviewOrderButtonClicked
+        reviewOrderButtonClicked,
+        revertPaymentInformationDone
     } = talonProps;
 
     const [, { addToast }] = useToasts();
 
     useEffect(() => {
         if (hasError) {
+            const message =
+                error && error.message
+                    ? error.message
```

Comment:
We don't want to be displaying backend-specific things to users like `GraphQL error:...`. Keep the original error message/error log please.

Reply:
Yeah, I asked the same question above. But one advantage of showing the error would be to let them know what exactly is the error because an error can be any one of [these](https://devdocs.magento.com/guides/v2.3/graphql/mutations/place-order.html#errors).

Reply:
Is `error.message` just the message string or the one with `GraphQL: <message>`? I just would prefer we strip off `GraphQL Error:` at least.

Reply:
It is the while unmodified message string from GQL. That's a nice idea to strip off stuff that we might not need but are we sure that this is the only way the error message will be sent to us?

Reply:
> are we sure that this is the only way the error message will be sent to us?

Good question for the GQL team - will the error format ever change?

Reply:
![image](https://user-images.githubusercontent.com/35203638/82585174-37e05780-9b5b-11ea-9687-787f376f41a8.png)

Reply:
Looks much better now.